app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    line = command.split(" ")
+
+    if len(line) != 3 or line[0] != "cp":
+        raise ValueError
```

Comment:
It's good that you've added input validation. However, when raising a `ValueError`, it's a good practice to include an error message to provide more context about what went wrong. Consider using `raise ValueError('Invalid command format')`.